docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
+BUILDDIR      = docbuild
```

Comment:
"build" or "docbuild": generally, "build" is the name used

Reply:
Yes, you are right. it was a resudual change of trying to make the build documentation available on BiblioMeter repository. I'll come back to "build".
Many thanks